### DISCUSSION TITLE
How can I hide the copy toast when I search ？

### DISCUSSION THREAD

**Discussion by gvnzhou** [sourcelair/xterm.js#4468] [Q&A] [answered]
Hi,
I want to hide the copy toast when I am using findNext API of **xterm-addon-search** :
![image](https://user-images.githubusercontent.com/13270798/230355258-fadadd72-9ee0-4643-805d-d9697a58b507.png)
At the same time, I found it isn't actually copied.

**Comment by jerch** [comment, marked as answer]
Are you sure you posted this question in the right repo? xterm.js does not draw any "copy toast".

Gonna close this, feel free to re-open if its indeed xterm.js related.
